custom_components/tahoma/const.py

```diff
@@ -41,6 +43,8 @@
     "Gate": "cover"
 }
 
+TAHOMA_TYPE_HEATING_SYSTEM = "HeatingSystem"
```

Comment:
By the way, one small comment for future PR's. What is this one used for? 

I don't see it in this PR, and I think it could be added to https://github.com/iMicknl/ha-tahoma/blob/add-device-class-to-sensors/custom_components/tahoma/const.py#L24 -> mapping `climate` in the end?

Reply:
oups... that slipped from the climate branch... I'm still quite bad at git...

Reply:
:D